modules/sandboxed-containers-deleting-catalog-subscription.adoc

```diff
@@ -0,0 +1,20 @@
+//Module included in the following assemblies:
+//
+// *uninstalling-sandboxed-containers.adoc
+
+[id="sandboxed-containers-deleting-catalog-subscription_{context}"]
+= Deleting the {sandboxed-containers-first} catalog subscription
```

Comment:
Should this be "Deleting the {sandboxed-containers-first} operator" for consistency with the CLI section?

Reply:
@zanetworker We can...but the intro sentence is: You can delete the OpenShift sandboxed containers catalog subscription and remove namespace access to the OpenShift sandboxed containers Operator.

Reply:
Can we change it to "You can delete the OpenShift sandboxed containers operators by deleting the catalog subscription and removing namespace access to the Operator?" 

Would this statement be technically correct @jensfr?